### PR TITLE
Dispatch Hello-World MWT Scale Test Jobs in Parallel

### PR DESCRIPTION
**Pull request by kaiwalyajoshi**
- Add support for threading to dispatch hello-world runs in parallel. This cuts down the running time of hello-world in MWT from hours to a few mins.
- Add support for batch cleanup of hello-world runs in MWT. This is a nice-to-have but useful.

### CODE REVIEW

File: frameworks/helloworld/tests/scale/test_scale.py [hunk 1]
```diff
@@ -18,41 +22,97 @@ def test_soak_load(service_name,
...
+    if min_index == -1 or max_index == -1:
+        deployment_list = ["{}-{}".format(scale_service_name, index) for index in range(0, int(service_count))]
+        min_index = 0
+        max_index = service_count
+    else:
+        deployment_list = ["{}-{}".format(scale_service_name, index) for index in range(min_index, max_index)]
+
+    sdk_security.install_enterprise_cli()
+
+    current = 0
```
Comment (takirala) [inline frameworks/helloworld/tests/scale/test_scale.py]: Does this code support the following use case:

1. Run the script once to launch 25 instances (0 to 24) of hw.
2. Script exits (may be some other person wants to takes over at a later point in time)
3. Run the script again to launch instances (25 to 49) of hw.

Comment (kaiwalyajoshi) [reply frameworks/helloworld/tests/scale/test_scale.py]: Yes.

File: frameworks/helloworld/tests/conftest.py [hunk 2]
```diff
@@ -16,6 +16,12 @@ def pytest_addoption(parser):
                      help="hello world service yml to use")
     parser.addoption("--service-name", action='store', default='hello-world',
                      help="custom service name to be used instead of default 'hello-world'")
+    parser.addoption('--min', action='store', default=-1,
+                     help='min jenkins index to start from (default: -1).')
```
Comment (kvish) [inline frameworks/helloworld/tests/conftest.py]: nit remove jenkins from the help text

Comment (kaiwalyajoshi) [reply frameworks/helloworld/tests/conftest.py]: whoops! good catch, many thanks!

**Pull request merged**